hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAsyncTableGetMultiThreaded.java

```diff
@@ -211,6 +210,11 @@ public String explainFailure() throws Exception {
       LOG.info("====== Move meta done ======");
       Thread.sleep(5000);
     }
+    BalancerDecisionRequest balancerDecisionRequest = new BalancerDecisionRequest();
+    balancerDecisionRequest.setLimit(2);
```

Comment:
Per other comments limit should be lifted into admin API not set per request type.